owasp-dependency-check-suppressions.xml

```diff
@@ -331,9 +331,13 @@
     <notes><![CDATA[
      file name: hadoop-*-3.3.1.jar
      ]]></notes>
-    <packageUrl regex="true">^pkg:maven/org\.apache\.hadoop/hadoop\-.*@.*$</packageUrl>
```

Comment:
why is this line removed?

Reply:
It was removed in the snippet used for reference